tools/provision/chocolatey/openssl.ps1

```diff
@@ -88,15 +93,20 @@ if (-not (Test-Path $zipFile)) {
   Invoke-WebRequest $url -OutFile "$zipFile"
 }
 
+$7z = (Get-Command '7z').Source
```

Comment:
For every invocation of `Get-Command` that you use, check the output, and ensure that we `-ErrorAction SilentlyContinue`